gitpod-io/gitpod
# [image-builder] Setup services based on baseserver

## Description
This PR makes image-builder use baseserver to set up its services

## How to test
- Ensure that image-builder starts
- Start a workspace

## Release Notes

```release-note
NONE
```
